code/modules/martial_arts/combos/sleeping_carp/crashing_kick.dm

```diff
@@ -0,0 +1,18 @@
+/datum/martial_combo/sleeping_carp/crashing_kick
+	name = "Crashing Waves Kick"
+	steps = list(MARTIAL_COMBO_STEP_HARM, MARTIAL_COMBO_STEP_GRAB)
+	explaination_text = "Kicks the target square in the chest. Knocks down."
+
+/datum/martial_combo/sleeping_carp/crashing_kick/perform_combo(mob/living/carbon/human/user, mob/living/target, datum/martial_art/MA)
+	if(!target.stat && !target.IsWeakened())
+		user.do_attack_animation(target, ATTACK_EFFECT_KICK)
+		target.visible_message("<span class='warning'>[user] kicks [target] in the chest!</span>", \
```

Comment:
```suggestion
		target.visible_message("<span class='warning'>[user] kicks [target] in the chest!</span>",
```
You don't need the `\` since it comes after a comma.